index.html

```diff
@@ -7,7 +7,50 @@
  </head>
  <body>
   <!--Шапка сайта-->
-  <header>Шапка сайта</header>
+  <header class="header">
+    <div class="header__wrapper">
+      <button class="header__logo" type="button" id="logo">
+        <img src="assets/icons/logo.svg" alt="ZEN"/>
+      </button>
+      <ul class="menu" id="menu">
+        <!-- Item 1 -->
+        <li class="menu__item">
+          <a href="#download">Download</a>
+        </li>
+        <!-- Item 2 -->
+        <li class="menu__item">
+          <a href="#warranty">Warranty</a>
+        </li>
+        <!-- Item 3 -->
+        <li class="menu__item">
+          <a href="#care">Care</a>
+        </li>
+        <!-- Item 4 -->
+        <li class="menu__item">
+          <a href="#cashback">Cashback</a>
+        </li>
+        <!-- Item 5 -->
+        <li class="menu__item">
+          <a href="#clients">Clients</a>
+        </li>
+      </ul>
+      <div class="header__actions">
+        <!-- Lang -->
+        <select class="header__lang" id="lang">
+          <option value="en">EN</option>
+          <option value="ru">RU</option>
+        </select>
+        <!-- Theme -->
+        <div class="header__theme" type="button" id="theme">
```

Comment:
please use tag <button> for theme (not <div>)

Reply:
fixed it, please check